common/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,7 @@
     <string name="auth_request">Authentication Requested</string>
     <string name="autoplay_video_summary">Autoplay Videos when lovelace dashboard is active. Enabling this setting may increase data usage unexpectedly, proceed with caution.</string>
     <string name="autoplay_video">Autoplay Videos</string>
-    <string name="background_access_disabled">Home Assistant does not have access to run in the background.  Click here to request permissions.</string>
+    <string name="background_access_disabled">Home Assistant does not have access to run in the background.  Without this permission the app will not be able to reliably send data back to your server.  Click here to request permissions.</string>
```

Comment:
This line uses double spaces between sentences, while (almost) all other lines with multiple sentences use a single space.